webrtc-stats.html

```diff
@@ -153,10 +199,9 @@
           (camelCase).
         </p>
         <p>
-          Names ending in "Id" (such as "datachannelId") are always of type DOMString and are used
-          to refer to other stats objects by the value of their "id" field; names ending in "Ids"
-          (such as "trackIds") are always of type sequence&lt;DOMString&gt;, and are used to refer
-          to a list of other stats objects.
+          Names ending in "Id" (such as "datachannelId") are always a <a>stats object
+          reference</a>; names ending in "Ids" (such as "trackIds") are always of type
```

Comment:
I think "stats object reference" would work with my definition, not with the one here.